server/data/migrations/20210113102811_session_table.js

```diff
@@ -0,0 +1,7 @@
+exports.up = (knex) =>
+  knex.schema.createTable('sessions', (table) => {
+    table.string('sid').notNullable();
+    table.json('sess').notNullable();
+    table.timestamp('expired').notNullable();
```

Comment:
Storage for the sessions ID, the Session as a JSON object, and the time the session expires